tracker/src/Tracker/Models/Interop/Pool.hs

```diff
@@ -0,0 +1,27 @@
+module Tracker.Models.Interop.Pool where
+
+import Tracker.Models.Interop.Wrappers
+
+import Data.Aeson   (FromJSON, ToJSON)
+import Ledger           
+import GHC.Generics (Generic)
+import RIO
+
+import ErgoDex.Amm.Orders
+import ErgoDex.Contracts.Types as Currencies
+import ErgoDex.State
+import ErgoDex.Amm.Pool
+import ErgoDex.Class
+import ErgoDex.Types
+
+data Pool = Pool
```

Comment:
why not ErgoDex.Amm.Pool?

Reply:
Cause I did an interop module for better json serialisation(`ErgoDex.Amm.Pool` contains models from plutus which json codecs are awful)